cli/smartcontract/smart_contract.go

```diff
@@ -54,7 +54,11 @@ var (
 	}
 	gasFlag = flags.Fixed8Flag{
 		Name:  "gas, g",
-		Usage: "gas to add to the transaction",
+		Usage: "network fee to add to the transaction (prioritizing it)",
+	}
+	sysGasFlag = flags.Fixed8Flag{
+		Name:  "sysgas, e",
```

Comment:
Why `e`?

Reply:
execution. `-s` is already occupied by
```
        cli.DurationFlag{
                Name:  "timeout, s",
                Usage: "Timeout for the operation (10 seconds by default)",
        },
```

Reply:
The help message needs to be synchronized too.

Reply:
Fixed.